src/Core/Domain/Product/Command/SetProductTagsCommand.php

```diff
@@ -45,7 +46,7 @@ class UpdateProductTagsCommand
     /**
      * @var LocalizedTags[]
      */
-    private $localizedTagsList;
+    private $localizedTagsList = [];
```

Comment:
I'm a bit confused in `SetAssociatedProductAttachmentsCommand` you remove the default value, but in this one you add one 😅  So what's the purpose and/or the convention in this kind of cases?

I may be wrong but since empty value is forbidden for both and array input is mandatory in the constructor, seems like the initial value will be erased anyway

Reply:
you are right, it is just a mistake, I think I put it before it had a dedicated command.
It is not needed anymore as this command cannot be contain empty array anymore